[2020-06-08 19:52:00,657] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_dag.source2 2020-06-08T14:13:36.673104+00:00 [queued]>
[2020-06-08 19:52:00,667] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: my_dag.source2 2020-06-08T14:13:36.673104+00:00 [queued]>
[2020-06-08 19:52:00,667] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-08 19:52:00,667] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-08 19:52:00,667] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-08 19:52:00,680] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): source2> on 2020-06-08T14:13:36.673104+00:00
[2020-06-08 19:52:00,682] {standard_task_runner.py:53} INFO - Started process 14466 to run task
[2020-06-08 19:52:00,744] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: my_dag.source2 2020-06-08T14:13:36.673104+00:00 [running]> nineleaps-ThinkPad-E470
[2020-06-08 19:52:00,816] {logging_mixin.py:112} WARNING - /home/nineleaps/airflow/dags/airflow_tasks.py:56: PendingDeprecationWarning: Client.dataset is deprecated and will be removed in a future version. Use a string like 'my_project.my_dataset' or a cloud.google.bigquery.DatasetReference object, instead.
  dataset_ref = client.dataset(DATA_SET_ID)
[2020-06-08 19:52:10,638] {logging_mixin.py:112} INFO - Loaded 37 rows into covid19:covid_data.
[2020-06-08 19:52:10,638] {python_operator.py:114} INFO - Done. Returned value was: 37.0
[2020-06-08 19:52:10,685] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=my_dag, task_id=source2, execution_date=20200608T141336, start_date=20200608T142200, end_date=20200608T142210
[2020-06-08 19:52:10,748] {logging_mixin.py:112} INFO - [2020-06-08 19:52:10,746] {local_task_job.py:167} WARNING - State of this instance has been externally set to success. Taking the poison pill.
[2020-06-08 19:52:10,781] {helpers.py:325} INFO - Sending 15 to GPID 14466
[2020-06-08 19:52:10,781] {helpers.py:291} INFO - Process psutil.Process(pid=14466, status='terminated') (14466) terminated with exit code 0
[2020-06-08 19:52:15,718] {logging_mixin.py:112} INFO - [2020-06-08 19:52:15,717] {local_task_job.py:103} INFO - Task exited with return code 0
